### PR TITLE
ui: display status in UI

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-elasticsearch

File: statics/js/components/status.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+var StatusComponent = {
+
+  name: 'Status',
+
+  mixins: [apiMixin, notificationMixin],
+
+  template: '\
+    <div class="status statustab">\
+      <panel title="Status" :collapsed="false">\
```
Comment (safchain) [inline statics/js/components/status.js]: I would make it collapsible, and I would set "Services status" as title

File: statics/index.html [hunk 2]
```diff
@@ -146,6 +149,9 @@
             <router-link v-if="service == 'Analyzer'" tag="li" to="/discovery">
               <a>Discovery</a>
             </router-link>
+            <router-link tag="li" to="/status">
```
Comment (lebauce) [inline statics/index.html]: I would prefer to have the tab at the top right (at the left of "Logout" for instance)

File: statics/js/components/status.js (hunk 1 above)
Comment (masco-enovance) [reply statics/js/components/status.js]: done

File: statics/index.html (hunk 2 above)
Comment (masco-enovance) [reply statics/index.html]: done

**Pull request merged**